### ISSUE TITLE
good first issue

### ISSUE THREAD

**Issue by BrokenArrow72** [BrokenArrow72/SeaBeesFirstLanding#1]
Yes can someone please help me with first my first code? Good first issue

**Comment by replyre** [comment]
what help do u need?

**Comment by abdull348** [comment]
@BrokenArrow72 Yes how can I help you?

**Comment by BrokenArrow72** [comment]
Yes I'm having an issue with any and all Android updates I'm attempting to
keep current. I am new to this so I am still in the learning process. Every
time I try to get a current update. It always says that my phone has a
specialized OS and to cont my administrator. Why is this. As far as I know
I am the administrator. Please help me understand this so I can fix this.
Thanks to all of you my Senior developers.

**Comment by MuhammadAbbasOfficial** [comment]
If none of the solutions work, performing a factory reset may help. This will remove all settings, apps, and management profiles from the device.
To factory reset, go to Settings > System > Reset Options > Erase All Data (Factory Reset). Make sure to back up any important data first.

**Comment by BrokenArrow72** [comment]
My platform is working fine, I have a bug that I can't seem to discover or
see. Not sure if it's a worm or a horse? Need some kind of confrontation
and help with the program. Is there anyone that can help or at least steer
me in the correct direction? Thanks for any assistance in this matter. In
debt to you. 236

**Comment by 01AdI** [comment]
is your phone given to you by a company?

**Comment by BrokenArrow72** [comment]
No it was not. It's a privately owned, no contract.
